### PR TITLE
Update WASM bindings

### PR DESCRIPTION
**Pull request by piotr-roslaniec**

### CODE REVIEW

File: nucypher-core-wasm/src/lib.rs [hunk 1]
```diff
@@ -112,6 +112,11 @@ impl MessageKit {
         Capsule::new(self.0.capsule)
     }
 
+    #[wasm_bindgen(method, getter)]
+    pub fn ciphertext(&self) -> Vec<u8> {
+        self.0.ciphertext.clone().to_vec()
```
Comment (fjarri) [inline nucypher-core-wasm/src/lib.rs:117]: Check if you need `clone()` here, I think `to_vec()` creates a copy already

File: nucypher-core/src/message_kit.rs [hunk 2]
```diff
@@ -17,7 +17,8 @@ pub struct MessageKit {
     /// Encapsulated symmetric key for this message.
     pub capsule: Capsule,
     #[serde(with = "serde_bytes")]
-    ciphertext: Box<[u8]>,
+    /// Ciphertext of the message.
+    pub ciphertext: Box<[u8]>,
```
Comment (fjarri) [inline nucypher-core/src/message_kit.rs:21]: Why do you need to expose it?

File: nucypher-core-wasm/src/lib.rs [hunk 3]
```diff
@@ -347,6 +358,20 @@ impl TreasureMap {
         Ok(serde_wasm_bindgen::to_value(&result)?)
     }
 
+    #[wasm_bindgen(js_name = makeRevocationOrders)]
+    pub fn make_revocation_orders(&self, signer: &Signer) -> Vec<JsValue> {
```
Comment (fjarri) [inline nucypher-core-wasm/src/lib.rs:362]: This should probably be made a method in the Rust part, and then exposed via bindings.

File: nucypher-core/src/revocation_order.rs [hunk 4]
```diff
@@ -13,7 +13,8 @@ use crate::versioning::{
 /// Represents a string used by characters to perform a revocation on a specific Ursula.
 #[derive(PartialEq, Debug, Serialize, Deserialize)]
 pub struct RevocationOrder {
-    staker_address: Address,
+    /// The address of the Ursula that is being revoked.
+    pub staker_address: Address,
```
Comment (fjarri) [inline nucypher-core/src/revocation_order.rs:17]: I guess we're finally doing something with revocation orders? Is it used anywhere yet, or is it just a preparation?

Also if it's exposed in JS, it needs to be exposed in Python bindings as well.

Comment (piotr-roslaniec) [reply nucypher-core/src/revocation_order.rs:17]: This is probably a bit premature given that IDK how revocations are going to work post-v6.0.0

File: nucypher-core/src/message_kit.rs (hunk 2 above)
Comment (piotr-roslaniec) [reply nucypher-core/src/message_kit.rs:21]: No reason, that was an accidental change. I'll revert it.

File: nucypher-core/src/hrac.rs [hunk 5]
```diff
@@ -37,6 +37,11 @@ impl HRAC {
         let (hrac, _rest): (GenericArray<u8, U16>, GenericArray<u8, _>) = digest.split();
         Self(hrac.into())
     }
+
+    /// Creates a new HRAC from an array of bytes.
+    pub fn from_bytes(bytes: [u8; 16]) -> Self {
```
Comment (fjarri) [inline nucypher-core/src/hrac.rs:42]: Sorry for nitpicking, but we already have `ProtocolObject::from_bytes()`, and I'd rather not have them mixed up. What do you think about making it an `impl From<[u8; 16]>` instead?

**Pull request merged**